services/storehost/storehost.go

```diff
@@ -1558,3 +1558,51 @@ func (t *StoreHost) RemoteReplicateExtent(tCtx thrift.Context, req *store.Remote
 
 	return nil
 }
+
+// ListExtents lists extents available on this storehost
+func (t *StoreHost) ListExtents(tCtx thrift.Context) (res *store.ListExtentsResult_, err error) {
+
+	t.m3Client.IncCounter(metrics.ListExtentsScope, metrics.StorageRequests)
+
+	sw := t.m3Client.StartTimer(metrics.ListExtentsScope, metrics.StorageLatencyTimer)
+	defer sw.Stop()
+
+	log := t.logger
+
+	log.Info("ListExtents()")
```

Comment:
this was following a pattern where every API in storehost logs on entry and exit. but i agree, we can make an exception for ListExtents since it is invoked only by 'cherami-admin' command.